src/transformers/models/auto/modeling_auto.py

```diff
@@ -428,6 +429,7 @@
         # Model for Object Detection mapping
         ("detr", "DetrForObjectDetection"),
         ("yolos", "YolosForObjectDetection"),
+        ("owlvit", "OwlViTForObjectDetection"),
```

Comment:
This will for sure cause issues in test_pipelines_object_detection, because it will test each model defined in the AutoModelForObjectDetection mapping, and as OWL-ViT is very different from DETR, it would require an update of the object detection pipeline. In the meantime, you can add this model to the "non auto configured" models in utils/check_repo.py (`make fixup` will tell you that)

So I would remove this model from the object detection mapping for now, perhaps we could create a "zero-shot object detection" pipeline later on, similar to the zero-shot image classification one.

cc @Narsil